### PR TITLE
update telemetry_autoscaling role

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

File: ci/telemetry_autoscaling.yml [hunk 1]
```diff
@@ -5,78 +5,10 @@
...
-          oc patch openstackversions openstack-galera-network-isolation --type json -p='[{"op": "replace", "path": "/spec/customContainerImages", "value": {} }]'
-      tags:
-        - never
-        - cleanup
+    - name: "Run Telemetry Autoscaling tests"
+      ansible.builtin.import_role:
+        name: telemetry_autoscaling
+      vars:
+        vars_path: "{{ vars_dir }}"
+      tags: autoscaling
```
Comment (elfiesmelfie) [inline ci/telemetry_autoscaling.yml:14]: Does Zuul support using tags?

File: roles/telemetry_autoscaling/tasks/post_teardown.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ignore_errors: true
+  register: output
+
+- name: Remove the stack if it exists
+  ansible.builtin.command:
+    cmd: |
+      oc rsh openstackclient openstack stack delete {{ stack_name }}
+  when: output.rc == 0
+
+- name: Check whether a stack already exists with the same name
```
Comment (elfiesmelfie) [inline roles/telemetry_autoscaling/tasks/post_teardown.yml:15]: The name of this should be updated. It is mis-represented currently.

```suggestion
- name: Wait until the stack is deleted
```

**Review by elfiesmelfie — CHANGES_REQUESTED** [review]

File: ci/telemetry_autoscaling.yml (hunk 1 above)
Comment (mgirgisf) [reply ci/telemetry_autoscaling.yml:14]: As far as I know yes zuul can use tags but this file is not being executed with zuul but with run_playbooks.yml so we can filter with --tag or --skip-tag for future integration with other tests

**Review by elfiesmelfie — APPROVED** [review]

**Comment by elfiesmelfie** [comment]
recheck

**Comment by mgirgisf** [comment]
recheck

**Comment by mgirgisf** [comment]
recheck

**Comment by elfiesmelfie** [comment]
recheck

**Comment by elfiesmelfie** [comment]
recheck

**Pull request merged**